### DISCUSSION TITLE
rAudio Bluetooth connection as receiver/sink not possible

### DISCUSSION THREAD

**Discussion by klassikmann** [rern/rAudio#101] [Bug Reports]
Since my last successful use to connect my smartphone to rAudio about midst December, pairing from the sender device is still possible now but no connection.

**Comment by ceejayemm** [comment]
@klassikmann 

Might not be relevant but have a look at the new 'Sampling 16bit 44.1kHz to receivers' option under the Bluetooth settings on the System page. It solved my problem with no sound on BT speakers from some album encodings as detailed in #12.

Might be a red herrings though.

Chris

**Reply by klassikmann** [reply]
Hi Chris,
sending to BT speakers is working as before. But the other way round, connecting to rAudio from my phone/tablet is no more possible.

**Comment by klassikmann** [comment]
@rern 
a) Do you confirm the problem?
b) Is there any log/debug info which I'm able to generate?

Additional observation: After successful pairing initiated by my smartphone, trying to connect to rAudio does nothing as if something is preventing a connection attempt on the smartphone. I reproduced this behaviour on a second device, a tablet which was also able to connect to re6 last month.

**Comment by klassikmann** [comment]
@rern 
I'm still try to establish the receiver/sink mode. My current conjecture is that the cause of the problem is on the Android side (see my last comment). What's the difference in the former successful pairing+connection in re6 and now using rAudio-1 (successful pairing, but no connection)? I see only one difference, namely the change in the bluetooth hostname (RuneAudio -> rAudio). 

Because my Android devices aren't rooted, I have no change to extend the investigation on the client side.

**Comment by klassikmann** [comment]
@rern 
The problem (pairing from smartphone to rAudio possible, but no connection) is solved, but I don't know why.
What I did:
- Testing with latest update (20210121) unsuccessful
- New sdcard with latest official image written
- Boot with new sdcard and update to 20210121
- pacman -Syu (btw. there is a new mpd (0.22.4)
- Pairing **and** connection works

Category changed to "Fixed issues"

**Comment by klassikmann** [comment]
The issue is back again.
Using rAudio as a BT receiver/sink does not work. Pairing from the sender device is possible now but no connection.
I didn't use the configuration this way for the last two months, so I'm unable to "bisect" the problem.

**Comment by EarsofTin** [comment]
I did a fresh install and update today for other reasons.  BT is working for me in both directions.  I do have a pair of headphones that I have never been able to connect but playing to a portable speaker right now.  Sometimes I have to "forget" a device, run the search again and pick the device from the new list.  So BT works for me but unfortunately everyone's operating under a unique set of conditions.

**Reply by klassikmann** [reply]
@EarsofTin 

> Sometimes I have to "forget" a device, run the search again and pick the device from the new list.
> So BT works for me but unfortunately everyone's operating under a unique set of conditions.

My experiences with BT in rAudio are similar. As I wrote at [https://github.com/rern/rAudio-1/discussions/101#discussioncomment-302638](url). Only under certain (unknown) conditions Bt as receiver/sink works. I'll come back to this topic after the next fresh install. Let's see what's happening.
